apps/nuxt3-ssr/components/content/ContentBlockIntro.vue

```diff
@@ -43,9 +45,35 @@ function onContactClick() {
           <BaseIcon name="external-link" class="inline mr-2" />{{ link }}
         </a>
       </div>
+      <SideModal
+        :show="showContactInformation"
+        :fullScreen="false"
+        :slideInRight="true"
+        @close="showContactInformation = false"
+        buttonAlignment="right"
+      >
+        <ContentBlock title="Contact">
+          Email:
```

Comment:
https://www.grammarly.com/blog/spelling-e-mail-email/#:~:text=E%2Dmail%20and%20email%20are,use%20the%20spelling%20it%20prescribes.

evt e-mail of E-mail